frontend/src/pages/clusterSettings/ClusterSettings.tsx

```diff
@@ -35,7 +35,7 @@ const ClusterSettings: React.FC = () => {
   const [userTrackingEnabled, setUserTrackingEnabled] = React.useState(false);
   const [cullerTimeout, setCullerTimeout] = React.useState(DEFAULT_CULLER_TIMEOUT);
   const { dashboardConfig } = useAppContext();
-  const modelServingEnabled = useIsAreaAvailable(SupportedArea.MODEL_SERVING);
+  const modelServingEnabled = useIsAreaAvailable(SupportedArea.MODEL_SERVING).status;
```

Comment:
Just realized this check here was incorrect and updated it, now it can really hide the platform settings section when the model serving feature is disabled. Also, verified the whole application. The model serving nav bar and routes are hidden. The custom serving runtime settings are hidden. The model serving section on the project details page is hidden.

<img width="1725" alt="Screenshot 2023-11-17 at 8 20 02 AM" src="https://github.com/opendatahub-io/odh-dashboard/assets/37624318/34748e36-0fba-4a9e-83ce-157d5deab2fa">

<img width="1725" alt="Screenshot 2023-11-17 at 8 18 44 AM" src="https://github.com/opendatahub-io/odh-dashboard/assets/37624318/38f8821e-c2b7-484b-be38-e2495a10c73d">

Reply:
Nice catch!